core/src/appleMain/kotlin/CentralManagerDelegate.kt

```diff
@@ -24,8 +27,8 @@ internal class CentralManagerDelegate : NSObject(), CBCentralManagerDelegateProt
     private val _onDisconnected = MutableSharedFlow<NSUUID>()
     internal val onDisconnected = _onDisconnected.asSharedFlow()
 
-    private val _state = MutableStateFlow<CBManagerState?>(null)
-    val state: Flow<CBManagerState> = _state.filterNotNull()
+    private val _state = MutableStateFlow<CBManagerState>(CBCentralManagerStateUnknown)
```

Comment:
nitpick: By providing the initial value, type inference is picking up the value and we don't need to explicitly list it anymore.

```suggestion
    private val _state = MutableStateFlow(CBCentralManagerStateUnknown)
```